android-base/src/main/java/io/github/droidkaigi/confsched2020/MainActivity.kt

```diff
@@ -93,8 +93,8 @@ class MainActivity : DaggerAppCompatActivity() {
         setupNavigation()
         setupStatusBarColors()
 
-        binding.drawerLayout.doOnApplyWindowInsets { _, insets, _ ->
-            binding.drawerLayout.setChildInsetsWorkAround(insets)
+        binding.drawerLayout.doOnApplyWindowInsets { view, insets, _ ->
+            (view as DrawerLayout).setChildInsetsWorkAround(insets)
         }
         binding.contentContainer.doOnApplyWindowInsets { _, insets, initialState ->
             binding.contentContainer.updatePadding(
```

Comment:
Are there any other similar issues that can be fixed at the same time?  🙏

Reply:
I will try to find it !